Disable comment trigger for Performance and Load Tests

magicgoody opened the pull request:
Reference PR https://github.com/apache/beam/pull/28915/files issue_comment removed 

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[![Build python source distribution and wheels](https://github.com/apache/beam/workflows/Build%20python%20source%20distribution%20and%20wheels/badge.svg?branch=master&event=schedule)](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[![Python tests](https://github.com/apache/beam/workflows/Python%20tests/badge.svg?branch=master&event=schedule)](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[![Java tests](https://github.com/apache/beam/workflows/Java%20Tests/badge.svg?branch=master&event=schedule)](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[![Go tests](https://github.com/apache/beam/workflows/Go%20tests/badge.svg?branch=master&event=schedule)](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

andreydevyatkin commented:
@Abacn Shall we disable comment trigger for Perf and Load tests only or for all workflows? Not sure if we actually need to disable it, but I see you did it here: https://github.com/apache/beam/pull/28915

Abacn commented:
> @Abacn Shall we disable comment trigger for Perf and Load tests only or for all workflows? Not sure if we actually need to disable it, but I see you started doing it here: #28915

Yes, for Perf and Load tests (and postcommits that have Jenkins counterpart). Until we replaced the comment trigger the precommit issue_comment triggered are kept because PR testing are most likely need them, and the number of precommit is tolerable for the queueing issue described in #28909

Abacn commented:
Notice that the tests that this PR involves are mostly newly added workflows after #28915

magicgoody commented:
> > @Abacn Shall we disable comment trigger for Perf and Load tests only or for all workflows? Not sure if we actually need to disable it, but I see you started doing it here: #28915
> 
> Yes, for Perf and Load tests (and postcommits that have Jenkins counterpart). Until we replaced the comment trigger the precommit issue_comment triggered are kept because PR testing are most likely need them, and the number of precommit is tolerable for the queueing issue described in #28909

Hello, thanks for checking
I also removed such workflow trigger for 4 PostCommit jobs where part of Jenkins is not yet disabled.

Abacn commented on .github/workflows/beam_PostCommit_Java_IO_Performance_Tests.yml:
```diff
@@ -20,8 +20,6 @@ on:
     tags: ['v*']
     branches: ['master', 'release-*']
     paths: ['it/google-cloud-platform/**','.github/workflows/beam_PostCommit_Java_IO_Performance_Tests.yml']
-  issue_comment:
```
these postcommit tests do not have jenkins counterpart, see https://github.com/apache/beam/issues/28426#issuecomment-1728293446

magicgoody replied:
Thanks for sharing the link, reverted changes back

Abacn commented on .github/workflows/beam_PostCommit_Python_Arm.yml:
```diff
@@ -18,8 +18,6 @@
 name: PostCommit Python Arm
 
 on:
-  issue_comment:
```
ARM tests do not have Jenkins counterpart

magicgoody replied:
Reverted back
